### DISCUSSION TITLE
A good vocoder can't be used with the trained mel models

### DISCUSSION THREAD

**Discussion by ikm565** [coqui-ai/TTS#2160] [General Q&A]
### Describe the bug

I use the hifi-gan I trained myself as the vocoder. For each mel model, the audio I get is shock noise, but I use this vocoder on the mel model I trained (based on [link1](https://github.com/NVIDIA/tacotron2)), and the speech is perfectly synthesized.
The vocoder was trained based on [hifi-gan](https://arxiv.org/abs/2010.05646) and [code-hifi-gan](https://github.com/jik876/hifi-gan)

### To Reproduce

train hifi-gan with [code-hifi-gan](https://github.com/jik876/hifi-gan) and use it as vocoder to synthesis voice wav

### Logs

```shell
all recent version
```

### Environment

```shell
{
    "CUDA": {
        "GPU": [],
        "available": false,
        "version": "11.6"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "1.13.0+cu116",
        "TTS": "0.8.0",
        "numpy": "1.21.6"
    },
    "System": {
        "OS": "Linux",
        "architecture": [
            "64bit",
            "ELF"
        ],
        "processor": "x86_64",
        "python": "3.8.13",
        "version": "#83~18.04.1-Ubuntu SMP Tue May 11 16:01:00 UTC 2021"
    }
}
```

**Comment by ikm565** [comment]
Has anyone had the same problem training or using hifi-gan?

**Comment by erogol** [comment]
you need to use the same audio parameters. Are you sure they match?
